CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
-cmake_minimum_required (VERSION 3.14.0)
-project (CubicServer)
+if(CMAKE_VERSION VERSION_GREATER_EQUAL 3.24.0)
+    # Suppress warning CMP0135
+    cmake_minimum_required(VERSION 3.24.0)
+else()
+    cmake_minimum_required(VERSION 3.14.0)
+endif()
```

Comment:
This policy does not exists for cmake 3.14, it has been introduce in 3.24

Reply:
But shouldn't that policy be set if the cmake version is a certain version ?
Then it would be the same than here but with a behaviour that would be the same on all versions if we set the policy to the old behaviour.